### DISCUSSION TITLE
Reactive Postgres and Metrics

### DISCUSSION THREAD

**Discussion by thackba** [quarkusio/quarkus#23385] [Q&A]
Hi. I have a short question. I tried the reactive PostgresSQL extension. During a test I got some errors about timeoutes. So I tried to find out how much connections the DB used. But I can not find a metric for that. Is there a way to get these metrics?

[a comment by an automated account is not shown]

**Comment by jaiprasad266** [comment]
I have a similar question , how do we get postgres metrics .
@thackba were you able to sort this ?

**Reply by thackba** [reply]
We removed the reactive postgres lib.

**Comment by brunobat** [comment]
@thackba @jaiprasad266 as far as I know, there is no support for metrics there, yet. Are you using Micrometer?

**Reply by jaiprasad266** [reply]
yes we are using micrometer
